multiapi-engine/src/main/java/com/sngular/api/generator/plugin/openapi/utils/MapperContentUtil.java

```diff
@@ -6,281 +6,312 @@
+                                                                                                                                                 specFile.getModelNameSuffix()))
+                                                                                                         .importList(getImportList(listSchema, modelPackage))
+                                                                                                         .schemaCombinator(schemaCombinatorType)
+                                                                                                         .fieldObjectList(listSchema)
+                                                                                                         .build());
+    return compositedSchemas;
   }
 
   private static List<SchemaFieldObject> getFields(
-      final Map<String, Schema> totalSchemas, final Schema<?> schema, final SpecFile specFile, final Map<String, SchemaObject> compositedSchemas, final List<String> antiLoopList) {
-    final var fieldObjectArrayList = new ArrayList<SchemaFieldObject>();
+      final Map<String, JsonNode> totalSchemas, final JsonNode schema, final SpecFile specFile,
+      final Map<String, SchemaObject> compositedSchemas, final List<String> antiLoopList) {
 
-    if (Objects.nonNull(schema.getProperties())) {
-      if (Objects.nonNull(schema.getAdditionalProperties())) {
-        schema.getProperties().forEach(
+    final var fieldObjectArrayList = new ArrayList<SchemaFieldObject>();
+    if (checkNotEmpty(schema,PROPERTIES)) {
+      if (checkNotEmpty(schema,ADDITIONAL_PROPERTIES)){
+        schema.get(PROPERTIES).fields().forEachRemaining(
             processProperties(totalSchemas, compositedSchemas, fieldObjectArrayList, specFile, schema, antiLoopList));
         fieldObjectArrayList.addAll(
-            processFieldObjectList("additionalProperties", null, schema, specFile, totalSchemas, compositedSchemas,
+            processFieldObjectList(ADDITIONAL_PROPERTIES, null, schema, specFile, totalSchemas, compositedSchemas,
                                    antiLoopList));
       } else {
         fieldObjectArrayList.addAll(
             processFieldObjectList(null, null, schema, specFile, totalSchemas, compositedSchemas, antiLoopList));
       }
-    } else if (ARRAY.equalsIgnoreCase(schema.getType())) {
-      final ArraySchema arraySchema = (ArraySchema) schema;
-      final String itemType = Objects.nonNull(arraySchema.getItems().get$ref()) ? getRef(arraySchema.getItems(), specFile) : arraySchema.getItems().getType();
+    } else if ((ARRAY.equalsIgnoreCase(getTextValue(schema,TYPE))) &&(checkNotEmpty(schema,ITEMS))){
+      final String itemType = Objects.nonNull(schema.get(ITEMS).get(REF)) ? MapperUtilCommon.getRef(schema.get(ITEMS), specFile.getModelNamePrefix(),
+                                                                                                    specFile.getModelNameSuffix()) : getTextValue(schema.get(ITEMS),TYPE);
```

Comment:
I think here you have some issues related with the style, so I suggest you to run the autoformatter.